### PR TITLE
Internal/docker creation

### PR DESCRIPTION
**Pull request by MajoBerger**
| Phases            | MP | MM  |  MB  | MR |   JM | Total  |
|-----------------|----:|----:|----:|-----:|-----:|-------:|
| ETA                  |  0  |  0  |    5 |     0 |      0 |        0 |
| Developing      |  0  |  0  |    16 |    0 |      0 |         0 |
| Review             |  0  |  0.5  |    0 |    0 |      0 |         0 |
| Total                |   -  |   -  |   -   |  -    |   -    |         0 |
| ETA est.             |      |      |       |       |         |         0 |
| ETA cust.           |   -  |   -  |   -  |   -   |   -     |        0 |

### CODE REVIEW

File: support/dspace_proxy.py [hunk 1]
```diff
@@ -31,5 +42,23 @@ def get(self, command, params=None, data=None):
...
+                log("Reauthenticate loop is over", Severity.DEBUG)
+                break
+            self.reauthenticate()
+            log("Reauthenticating", Severity.DEBUG)
+
 
 rest_proxy = DspaceRESTProxy()
+reauth = threading.Thread(target=rest_proxy.reauthenticate_loop, daemon=True)
+stopevent = threading.Event()
+reauth.start()
```
Comment (milanmajchrak) [inline support/dspace_proxy.py:59]: This reauthentication is solved in the `client.py` Please turn off this reauthentication by the cfg

File: test/metasharev2_format/__init__.py
Comment (milanmajchrak) [inline test/metasharev2_format/__init__.py:1]: Do we need this empty file?

**Review by milanmajchrak — CHANGES_REQUESTED** [review]

File: support/dspace_proxy.py (hunk 1 above)
Comment (MajoBerger) [reply support/dspace_proxy.py]: done. Please merge

File: test/metasharev2_format/__init__.py
Comment (MajoBerger) [reply test/metasharev2_format/__init__.py:1]: there are MANY MANY empty __init__.py files. They are (or used to be) necessary to consider folder a python package. I suggest keeping them

**Review by milanmajchrak — APPROVED** [review]

**Review by mrovnanik — APPROVED** [review]

**Comment by MajoBerger** [comment]
Way too old, too many changes to merge now

**Pull request closed without merging**